apps/src/templates/sectionProgress/detail/VirtualizedDetailView.jsx

```diff
@@ -251,10 +250,7 @@ class VirtualizedDetailView extends Component {
     // Add 1 to account for the student name column
     const columnCount = scriptData.stages.length + 1;
     // Calculate height based on the number of rows
-    const tableHeightFromRowCount =
-      ROW_HEIGHT * rowCount + LAST_ROW_MARGIN_HEIGHT;
-    // Use a 'maxHeight' of 680 for when there are many rows
-    const tableHeight = Math.min(tableHeightFromRowCount, MAX_TABLE_SIZE);
+    const tableHeight = ROW_HEIGHT * rowCount + LAST_ROW_MARGIN_HEIGHT;
```

Comment:
Would we need to make a similar change to improve performance for horizontal scrolling?

Reply:
No such change is required, there are no horizontal scrolling performance problems at this time